feat(content): Add new human merchant hails about mining asteroids (#9769)
endless-sky/endless-sky: data/human/hails.txt

		`Apparently they want me to pay 5% of my paycheck for union dues! I'm gonna keep it. My boss is a good guy; he'll look out for me.`
		`I saved up and bought extra bunk space so my wife could come on trips with me, and now she says "we need to talk." That can't be a good sign, can it?`

phrase "friendly civilian"
	word
		"If you want to "
		"To "
	word
		"make"
		"turn"
		"earn"
		"get"
	word
		" "
	word
		"a profit"
		"some credits"
		"good money"
		"a good turnover"
		"a living"
	word
		" "
	word
		"while mining"
		"as an asteroid miner"
		"mining asteroids"
		"in the asteroid mining business"
	word
		", "
	phrase
		"miner advice: outfit"
		"miner advice: mineral"
		"miner advice: ship"
		"miner advice: other"
	word

> 		"."

Reviewer: More diversity here might be nice.

```suggestion
		"." 3
		"!"
```

		"you should learn to "
		"you're better off to "
		"it's best to "
		""
	word
		"avoid mining "
		"avoid picking up "
		"avoid harvesting "
		"avoid "
		"dump "
		"jettison "
	word
		"silicon"
		"iron"
		"lead"
		"aluminum"
		"silicon ore"
		"iron ore"
		"lead ore"
		"aluminum ore"
		"minerals like silicon"
		"minerals like iron"
		"minerals like lead"
		"minerals like aluminum"
		"ore like silicon"
		"ore like iron"
		"ore like lead"
		"ore like aluminum"
	word
		" and mine other asteroids"
		" and find more valuable minerals"
		" so you have space for more profitable minerals"
		" because they don't sell for very much"

phrase "miner advice: ship"
	word
		"you'll want to"
		"it's a good idea to"
		"it's advisable to"
		"the best thing you can do is"
	word
		" "
	word
		"expand your fleet with"
		"purchase"
		"take out a mortgage for"
		"buy"
		"go to the shipyard and get"
		"get"
	word
		" a "
	word
		"specialized mining ship like a Sunder"
		"ship fit for mining, like a Sunder"
		"Syndicate Sunder"
		"Sunder and two mining drones"
		"Sunder mining vessel"
		"good mining ship, like a Sunder"
		"versatile ship like a Headhunter"
		"multi-purpose ship like a Headhunter"

> 		"carrier like an Aerie. You can use the fighters to mine for you"

Reviewer: Surely the Scout isn't getting enough attention here. Maybe something like:

```suggestion
		"carrier like an Aerie. You can use the fighters to mine for you"
		"nice sturdy ship like a Scout. You can fly it to the back of beyond, then fill its cargo bay with your haul"
```

Author: Maybe I should update the mining fleet with Clippers and Scouts.

		"ship with a large cargo hold"
		"good freighter that can fit mining laser turrets"
		"ship that can defend itself from pirates"
		"powerful escort ship that can scare off pirates"
		"ship that can fit loads of mining lasers"
		"Behemoth loaded with mining laser turrets"

phrase "miner advice: other"
	word
		"you should stick to patrolled systems so the pirates don't blow you up"
		"you should stick to patrolled systems so you won't get extorted by pirates"
		"you should avoid patrolled systems as they don't tend to have rare minerals"
		"you should learn to recognize what asteroids are worthless or valuable"
		"keep in mind that mining in systems frequented by pirates may be risky, but they often have good minerals"
		"consider bribing pirates to leave you alone in systems with valuable asteroids"

phrase "friendly civilian"
	word
		"I'm trying to improve my piloting skills without risking my life too much, so I've been mining asteroids."
		"Mining can be a good way to improve your piloting skills."

> 		"Mining is a safe career option for new Sparrow pilots."

Reviewer: Not necessary, of course, but it could help to emphasize that there aren't many safe options for Sparrow pilots.

```suggestion
		"Mining is the only safe career option for new Sparrow pilots."
```

		`Apparently they want me to pay 5% of my paycheck for union dues! I'm gonna keep it. My boss is a good guy; he'll look out for me.`
		`I saved up and bought extra bunk space so my wife could come on trips with me, and now she says "we need to talk." That can't be a good sign, can it?`

phrase "friendly civilian"
	word
		"If you want to "
		"To "
	word
		"make"
		"turn"
		"earn"
		"get"
	word
		" "
	word
		"a profit"
		"some credits"
		"good money"
		"a good turnover"
		"a living"
	word
		" "
	word
		"while mining"
		"as an asteroid miner"
		"mining asteroids"
		"in the asteroid mining business"
	word
		", "
	phrase
		"miner advice: outfit"
		"miner advice: mineral"
		"miner advice: ship"
		"miner advice: other"
	word
		"." 3
		"!"
	replace
		"turn some" "turn over some"
		"turn good" "turn over good"
		"turn a good turnover" "make a good turnover"

> 		"%I" "i"

Reviewer 2: Not seeing where this would be utilized. Was this leftover from a specific `word` that was removed?

Author: Yeah, these hails went through a lot of iterations. I'll remove it.

> 		"%I" "i"

Author: ```suggestion
```

		"get a living" "make a living"
		"turn a living" "make a living"

phrase "miner advice: outfit"
	word
		"you need"
		"it's a good idea to invest in"
		"you should get"
		"it's sensible to get"
		"get"
		"take out a loan to buy"
		"go to an outfitter and invest in"
		"you can't go wrong with"
		"the best thing to do is to install"
	word
		" some "
		" a good set of "
	word
		"asteroid scanners so you can detect and target asteroids more easily"
		"asteroid scanners. They make it much easier to target asteroids"
		"asteroid scanners. You won't regret the purchase"
		"asteroid scanners. If you don't know your way around asteroids, these will highlight and analyze their mineral composition"
		"asteroid scanners" 4
		"mining lasers. They're faster for mining than other lasers"
		"mining lasers. They're specially designed to break apart asteroids rapidly"
		"mining lasers. They yield you more minerals per asteroid"

> 		"mining lasers. Your mineral yield will go up noticibly with a few of those"

Reviewer 2: ```suggestion
		"mining lasers. Your mineral yield will go up noticeably with a few of those"
```

phrase "miner advice: other"
	word
		"you should stick to patrolled systems so the pirates don't blow you up"
		"you should stick to patrolled systems so you won't get extorted by pirates"
		"you should avoid patrolled systems as they don't tend to have rare minerals"
		"you should learn to recognize what asteroids are worthless or expensive"
		"keep in mind that mining in systems frequented by pirates may be risky, but they often have good minerals"
		"consider bribing pirates to leave you alone in systems with valuable asteroids"

phrase "friendly civilian"
	word
		"I'm trying to improve my piloting skills without risking my life too much, so I've been mining asteroids."
		"Mining can be a good way to improve your piloting skills."
		"Mining is a safe career option for new Sparrow pilots."
		"Asteroid miners should be better appreciated by other captains. It takes a lot of effort to destroy asteroids and just as much effort to scoop them up into your hold."
		"I'm sick and tired of other ships stealing the minerals I've mined before I can get to them."

phrase "friendly civilian"
	word
		"I find that %"
		"Often, %"
		"While difficult, %"

> 		"It looks difficult, but %"

Reviewer 3: ```suggestion
		"It looks difficult, but %"
		""
```
Isn't this why you have the replacements?

Author: Oops, I must have removed that by accident.

	word
		"Mining"
		"Harvesting"
	word
		" asteroids "
	word
		"can be "
		"is "
	word
		"surprisingly "
		"quite "
		"very "
	word
		"profitable"
		"fun"
		"relaxing"
		"worthwhile"
		"meditative"
	word
		" and "
	word
		"profitable"
		"fun"
		"relaxing"
		"worthwhile"
		"meditative"
	word
		"."
	replace
		"%M" "m"
		"%H" "h"
		"profitable and profitable" "profitable"
		"fun and fun" "fun"
		"relaxing and relaxing" "relaxing"
		"worthwhile and worthwhile" "worthwhile"
		"meditative and meditative" "meditative"

phrase "friendly civilian"
	word
		"It's becoming harder"
		"It used to be much easier"
		"I wish it was easier"
		"Once it was quite easy"
		"Once it was easier for captains"
	word
		" to work as "
		" to be "
	word
		"an asteroid miner"
		"a miner"
		"an independent miner"
	word
		"."
		"..."
	word
		"" 3
		" Pirate extortion fees are becoming just another business expense."

> 		" The pirate fleets get bolder and more aggressive every day, but the Navy only patrols in the Paradise Worlds."

Reviewer 3: I feel like these don't line up well with the FW campaign reducing piracy.

Author: Just this one? I can remove it.
```suggestion
```

> 		" The pirate fleets get bolder and more aggressive every day, but the Navy only patrols in the Paradise Worlds."
> 		" Pirates used to be few and far between, but now their warships terrorize the more profitable mining systems."
> 		" I used to have faith in the Navy's protection, but now I'm starting to lose count the number of times pirates have extorted me."

Reviewer 3: ```suggestion
```
These three, I think. The rest are fine even after FW.

Author: I do hope someday we'll have hails that can be adjusted with events, but I'll remove these anyway.

		`Apparently they want me to pay 5% of my paycheck for union dues! I'm gonna keep it. My boss is a good guy; he'll look out for me.`
		`I saved up and bought extra bunk space so my wife could come on trips with me, and now she says "we need to talk." That can't be a good sign, can it?`

phrase "friendly civilian"
	word
		"If you want to "
		"To "
	word
		"make"

> 		"turn"

Reviewer 4: ```suggestion
		"earn"
```

earn gives more variety, since you use "turnover" below. Gotta change the replacements too, if going this route.

You could also just add it to the list.

```suggestion
		"turn"
		"earn"
```

		"get"
	word
		" "
	word
		"a profit"
		"some credits"
		"good money"
		"a good turnover"
		"a living"
	word
		" "
	word
		"while mining"
		"as an asteroid miner"
		"mining asteroids"
		"in the asteroid mining business"
	word
		", "
	phrase
		"miner advice: outfit"
		"miner advice: mineral"
		"miner advice: ship"
		"miner advice: other"
	word
		"."
	replace
		"turn some" "turn over some"
		"turn good" "turn over good"
		"turn a good turnover" "make a good turnover"
		"%I" "i"
		"get a living" "make a living"
		"turn a living" "make a living"

phrase "miner advice: outfit"
	word
		"you need"
		"it's a good idea to invest in"
		"you should get"
		"it's sensible to get"
		"get"
		"take out a loan to buy"
		"go to an outfitter and invest in"
		"you can't go wrong with"
		"the best thing to do is to install"
	word
		" some "
		" a good set of "
	word
		"asteroid scanners so you can detect and target asteroids more easily"
		"asteroid scanners. It makes it much easier to target asteroids"
		"asteroid scanners. You won't regret the purchase"

> 		"asteroid scanners. If you don't know your way around asteroids, one of these will highlight and analyze them for their mineral composition"

Reviewer 4: ```suggestion
		"asteroid scanners. If you don't know your way around asteroids, these will highlight and analyze their mineral composition"
```
Making the plurals agree.
Bonus: shorter hail

		"asteroid scanners" 4

> 		"mining lasers. They're faster than other lasers for mining"

Reviewer 4: ```suggestion
		"mining lasers. They're faster for mining than other lasers"
```
Soft suggestion

		"heavy lasers" 2
		"tractor beams so pirates and rival miners can't steal your hard-earned ore"
		"tractor beams, which makes ore collection just that little bit easier"
		"tractor beams so picking up ore takes less effort"
		"tractor beams" 3

phrase "miner advice: mineral"
	word
		"you've got to "
		"you need to look around and "
		"you have to "
		""
	word
		"mine "
		"target "
		"go for "
		"find systems with "
		"look for "
	word
		"the most valuable minerals"
		"the most valuable asteroids"
		"rare asteroids"
		"uranium"
		"platinum"
		"gold"
		"silver"
		"tungsten"
		"neodymium"
		"uranium asteroids"
		"platinum asteroids"
		"gold asteroids"
		"silver asteroids"
		"tungsten asteroids"
		"neodymium asteroids"
		"minerals like uranium"
		"minerals like platinum"
		"minerals like gold"
		"minerals like silver"
		"minerals like tungsten"
		"minerals like neodymium"

phrase "miner advice: mineral"
	word
		"you've got to "
		"you should "
		"you should learn to "
		"you're better off to "
		"it's best to "
		""
	word
		"avoid mining "
		"avoid picking up "
		"avoid harvesting "
		"avoid "
		"dump "
		"jettison "
	word
		"silicon"
		"iron"
		"lead"

> 		"aluminum"

Reviewer 4: ```suggestion
```
aluminum is based

Author: aluminium ship hulls >: (

		"silicon ore"
		"iron ore"
		"lead ore"

> 		"aluminum ore"

Reviewer 4: ```suggestion
```

		"minerals like silicon"
		"minerals like iron"
		"minerals like lead"

> 		"minerals like aluminum"

Reviewer 4: ```suggestion
```

		"ore like silicon"
		"ore like iron"
		"ore like lead"

> 		"ore like aluminum"

Reviewer 4: ```suggestion
```

	word
		" and mine other asteroids"
		" and find more valuable minerals"
		" so you have space for more profitable minerals"
		" because they don't sell for very much"

phrase "miner advice: ship"
	word
		"you'll want to"
		"it's a good idea to"
		"it's advisable to"

> 		"the best thing you can do is to"

Reviewer 4: ```suggestion
		"the best thing you can do is"
```

	word
		" "
	word
		"expand your fleet with"
		"purchase"
		"take out a mortgage for"
		"buy"
		"go to the shipyard and get"
		"get"
	word
		" a "
	word
		"specialized mining ship like a Sunder"
		"ship fit for mining, like a Sunder"
		"Syndicate Sunder"
		"Sunder and two mining drones"
		"Sunder mining vessel"
		"good mining ship, like a Sunder"
		"defensible ship like a Headhunter"

> 		"multi-purpose ship like a Headhunter"

Reviewer 4: ```suggestion
		"multi-purpose ship like a Clipper"
```

Not enough love for Clippers imo.

Author: I disagree, Clippers are the meta.

		"defensible ship like an Argosy"
		"versatile ship like an Argosy"
		"carrier like an Aerie. You can use the fighters to mine for you"
		"squadron of Furies with mining lasers"
		"ship with a large cargo hold"
		"good freighter that can fit mining laser turrets"
		"ship that can defend itself from pirates"
		"powerful escort ship that can scare off pirates"
		"ship that can fit loads of mining lasers"
		"Behemoth loaded with mining laser turrets"

phrase "miner advice: other"
	word
		"you should stick to patrolled systems so the pirates don't blow you up"
		"you should stick to patrolled systems so you won't get extorted by pirates"
		"you should avoid patrolled systems as they don't tend to have rare minerals"
		"you should learn to recognize what asteroids are worthless or expensive"

> 		"mining in systems frequented by pirates may be risky, but they often have good minerals"

Reviewer 4: ```suggestion
		"keep in mind that mining in systems frequented by pirates may be risky, but they often have good minerals"
		"consider bribing pirates to leave you alone in systems with valuable asteroids"
```
Tossing in a pirate bribery plug as well

phrase "friendly civilian"
	word
		"I'm trying to improve my piloting skills without risking my life too much, so I've been mining asteroids."
		"Mining can be a good way to improve your piloting skills."

> 		"Mining is a safe career option for new interceptor pilots."

Reviewer 4: ```suggestion
		"Mining is a safe career option for new Sparrow pilots."
```
Little more on the nose, to help out beginners suffering through the Sparrow route.

> 		"Asteroid miners should be better appreciated by other captains. It takes a lot of effort to destroy asteroids, and just as much effort to scoop them up into your hold."

Reviewer 4: ```suggestion
		"Asteroid miners should be better appreciated by other captains. It takes a lot of effort to destroy asteroids and just as much effort to scoop them up into your hold."
```

		" some "
		" a good set of "
	word
		"asteroid scanners so you can detect and target asteroids more easily"
		"asteroid scanners. They make it much easier to target asteroids"
		"asteroid scanners. You won't regret the purchase"
		"asteroid scanners. If you don't know your way around asteroids, these will highlight and analyze their mineral composition"
		"asteroid scanners" 4
		"mining lasers. They're faster for mining than other lasers"
		"mining lasers. They're specially designed to break apart asteroids rapidly"
		"mining lasers. They yield you more minerals per asteroid"
		"mining lasers. Your mineral yield will go up noticibly with a few of those"
		"mining lasers. They're more precise than other lasers and preserve more minerals when the asteroid breaks up"
		"mining lasers so you can finish your mining run before the pirates get to you"
		"mining lasers" 5
		"heavy lasers. Their high range makes it easier to destroy asteroids, and you can also use them to scare off pirates"
		"heavy lasers. Because they have a greater damage output, they're faster at destroying asteroids"
		"heavy lasers" 2
		"tractor beams so pirates and rival miners can't steal your hard-earned ore"

> 		"tractor beams, which make ore collection just that little bit easier"

Author: ```suggestion
		"tractor beams. It makes ore collection just that little bit easier"
```

		""
	word
		"avoid mining "
		"avoid picking up "
		"avoid harvesting "
		"avoid "
		"dump "
		"jettison "
	word
		"silicon"
		"iron"
		"lead"
		"aluminum"
		"silicon ore"
		"iron ore"
		"lead ore"
		"aluminum ore"
		"minerals like silicon"
		"minerals like iron"
		"minerals like lead"
		"minerals like aluminum"
		"ore like silicon"
		"ore like iron"
		"ore like lead"
		"ore like aluminum"
	word
		" and mine other asteroids"
		" and find more valuable minerals"
		" so you have space for more profitable minerals"
		" because they don't sell for very much"

phrase "miner advice: ship"
	word
		"you'll want to"
		"it's a good idea to"
		"it's advisable to"
		"the best thing you can do is"
	word
		" "
	word
		"expand your fleet with"
		"purchase"
		"take out a mortgage for"
		"buy"
		"go to the shipyard and get"
		"get"
	word
		" a "
	word
		"specialized mining ship like a Sunder"
		"ship fit for mining, like a Sunder"
		"Syndicate Sunder"
		"Sunder and two mining drones"
		"Sunder mining vessel"
		"good mining ship, like a Sunder"
		"defensible ship like a Headhunter"
		"multi-purpose ship like a Headhunter"
		"defensible ship like an Argosy"
		"versatile ship like an Argosy"

> 		"carrier like an Aerie. You can use the fighters to mine for you"

Reviewer 5: Was this behavior actually merged yet? I can't recall, but I thought at one point it was still bound up in the other fighter behaviors PR?

Author: It should work, the Sunder and its Mining Drones also depend on it and they were implemented in 0.10.5.

	word
		"I'm trying to improve my piloting skills without risking my life too much, so I've been mining asteroids."
		"Mining can be a good way to improve your piloting skills."

> 		"Mining is the only safe career option for new Sparrow pilots."

Reviewer 3: ~~Technically, you can still make a profit by hauling cargo and passengers with a Sparrow, it's just painfully slow.~~
